### PR TITLE
Adds recipe for openSMILE tested for linux build [skip appveyor]

### PR DESCRIPTION
**Pull request by stevecassidy**
Recipe for the openSMILE speech processing toolkit.  I've only tried to configure this to build on linux, mac may work. Windows might be possible as there is a windows build script.

### CODE REVIEW

File: recipes/opensmile/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://audeering.com/download/1318/
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: sh buildStandalone.sh -p {{PREFIX}}
```
Comment (jakirkham) [inline recipes/opensmile/meta.yaml]: Could you please add ` || { cat config.log; exit 1; }` temporarily? Would like to get the error information from configure as it is unclear why it is failing?

**Comment by stevecassidy** [comment]
@jakirkham thanks, added the line and pushed, waiting for travis to complete. Also wonder why circleci is failing even to find the run_docker_build.sh script.

**Comment by jakirkham** [comment]
Can you also please merge with `master`? That should fix the missing file error on CircleCI.

**Comment by stevecassidy** [comment]
@jakirkham a better outcome but failing on osx, the build script assumes that 'libtoolize' is called 'glibtoolize' on osx, I think that isn't the case for the conda-forge package, will try to patch the build script to just use libtoolize.

File: recipes/opensmile/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://audeering.com/download/1318/
+  sha256: {{ sha256 }}
+  patches:
+    - autogen.patch  # fix libtoolize on osx
+
+build:
+  number: 0
+  skip: True  # [win or osx]
```
Comment (jakirkham) [inline recipes/opensmile/meta.yaml]: Should macOS be skipped if we are patching it to build on macOS?

**Comment by jakirkham** [comment]
Could try setting the `LIBTOOL` environment variable in our build scripts instead of patching. Alternatively could try adding a temporary symlink between `libtool` and `glibtool` in `$PREFIX/bin`.

**Comment by stevecassidy** [comment]
@jakirkham I don't know what to do here, Travis just seems to be cancelling the build and there is no way to skip it that I can see.  I'd like to at least get the linux64 build in place so I can move on from this. Any help appreciated.

**Comment by stevecassidy** [comment]
@jakirkham now that the osx builds are running, the problem seems to be missing librt:

```
configure:3428: clang -O2 -mfpmath=sse -msse2 -D__STATIC_LINK -DOPENSMILE_BUILD -DBUILD_SVMSMO -DBUILD_SVMSMO -DBUILD_LIBSVM -DBUILD_RNN -DBUILD_WITHOUT_EXPERIMENTAL -lrt -lm -lpthread -lc conftest.c  >&5
**ld: library not found for -lrt**
```
the -lrt doesn't get inserted when I build on my local osx so not clear how it's arising (can't find the string in the sources so it must be some artifact of autoconf etc.)

Unless you have any ideas I'll just disable the osx build for this package.

**Comment by stevecassidy** [comment]
@jakirkham just wondering if this can be finalised. Thanks.

**Pull request closed without merging**